core/src/devicebrowser.cpp

```diff
@@ -136,16 +141,35 @@ void DeviceBrowser::prevDevice()
 
 void DeviceBrowser::forwardRequestDeviceWithDirection()
 {
-    QString id = QObject::sender()->property(devBrowserId).toString();
-    int idx = getIndexOfId(id);
-    int direction = currentIdx - idx;
-    Q_EMIT requestDevice(id, direction);
+	QString id = QObject::sender()->property(devBrowserId).toString();
+	int idx = getIndexOfId(id);
+	int direction = currentIdx - idx;
+	Q_EMIT requestDevice(id, direction);
 }
 
 void DeviceBrowser::updateSelectedDeviceIdx(QString k)
 {	
 	int prevIdx = currentIdx; // local, just for debug	
 	currentIdx = getIndexOfId(k);
+
+	QWidget* prevDevice = getDeviceWidgetFor(getIdOfIndex(prevIdx));
+	QWidget* currentDevice = getDeviceWidgetFor(getIdOfIndex(currentIdx));
+
+	if (currentDevice == ui->btnHome)
+		currentDevice = ui->containerHome;
```

Comment:
doesnt' 
	connect(ui->btnHome, &QPushButton::toggled,this, [=](bool b) { ui->btnHome->setDynamicProperty(selected, b) } );

same for ui->btnAdd ?